### PR TITLE
CANS-946 - Remove RWD Dependency

### PR DESCRIPTION
**Pull request by wmitchell-cambria**
## JIRA

[CANS-946 - Replace React Wood Duck headers with Design System headers from Component Library](https://osi-cwds.atlassian.net/browse/cans-946)

## Description

This removes the RWD dependency, and attempts to adjust styles across the application to deal with that. Shout out to the whole team on this one... it was a huge group effort to verify that all styles were working correctly (and fixing the ones that weren't).

We still have some last cleanup to do, but I wanted to make this PR to get acceptance tests running as we go.

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/AssessmentHelper.js [hunk 1]
```diff
@@ -200,7 +200,7 @@ export const caregiverWarning = (
 )
 
 export const completeTip = (
-  <Typography variant="headline" className={'submit-validation-message'}>
+  <Typography variant="body1" className={'submit-validation-message'}>
```
Comment (abelayt) [inline app/javascript/Application/components/Assessment/AssessmentHelper.js]: We should be getting rid of material-ui whenever possible and in this case no need to use Typography.

**Comment by wmitchell-cambria** [comment]
retest this please

**Comment by wmitchell-cambria** [comment]
retest this please

**Comment by wmitchell-cambria** [comment]
retest this please

**Pull request merged**